tests/test_ciffilereaderwriter.py

```diff
@@ -25,8 +28,8 @@ def read_pos_trajectory(self, stream, precision=None):
         reader = garnett.reader.PosFileReader(precision=precision)
         return reader.read(stream)
 
-    def read_cif_trajectory(self, stream, precision=None):
-        reader = garnett.reader.CifFileReader(precision=precision)
+    def read_cif_trajectory(self, stream, precision=None, **kwargs):
+        reader = garnett.reader.CifFileReader(precision=precision, **kwargs)
```

Comment:
I'd look at removing the specific argument here for `precision` and letting it all pass through as `**kwargs`. If that's not a major change, I think it would improve clarity (handling precision and tolerance similarly).